tests/unit_tests.py

```diff
@@ -41,7 +41,7 @@
 import _pytest.config
 original_parse_setoption = _pytest.config.Parser.parse_setoption
 _pytest.config.Parser.parse_setoption = \
-    lambda *args: original_parse_setoption(*args) or [os.environ['TESTS_DIR']]
+    lambda *args, **kwargs: original_parse_setoption(*args, **kwargs) or [os.environ['TESTS_DIR']]
```

Comment:
You should keep one line up to 80 characters, but this line is now longer than that. Can you break the line so that it fits in 80 chars? e.g.

```python
_pytest.config.Parser.parse_setoption = (lambda *args, **kwargs:
    original_parse_setoption(*args, **kwargs) or [os.environ['TESTS_DIR']])
```

You can break in the middle of the expression if it is enclosed with ( ).

See this style guide for rules we should follow e.g., the 80 character limit:
https://google.github.io/styleguide/pyguide.html